kujira/assetlist.json

```diff
@@ -2462,6 +2462,35 @@
           }
         }
       ]
+    },
+    {
+      "description": "The token of MCTest",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+          "exponent": 0
+        },
+        {
+          "denom": "mc-test",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira1mn2pgkggquqm2m44zqlaq29jxwtad53qkv4xy4/MCTest",
+      "name": "MCTEST",
+      "display": "mctest",
```

Comment:
This denom must be defined as a unit but is not. I'd recommend removing the dash '-' in "mc-test" above, or else adding it here.

Reply:
fixed